src/cljs/athens/events.cljs

```diff
@@ -490,6 +490,31 @@
                                   :block/children reindex}]]]
           [:dispatch [:editing/uid new-uid]]]}))
 
+(defn split-block-to-children
+  "Takes a block uid, its value, and the index to split the value string.
+  It sets the value of the block to the head of (subs val 0 index)
+  It then creates a new child block with the tail of the string set as its value and sets editing to that block."
+  [uid val index]
+  (let [block (db/get-block [:block/uid uid])
+        head (subs val 0 index)
+        tail (subs val index)
+        new-uid (gen-block-uid)
+        new-block {:db/id        -1
+                   :block/order  (inc (:block/order block))
+                   :block/uid    new-uid
+                   :block/open   true
+                   :block/string tail}
+        reindex (->> (inc-after (:db/id block) (:block/order block))
+                     (concat [new-block]))]
+    {:fx [[:dispatch [:transact [{:db/id (:db/id block) :block/string head :edit/time (now-ts)}
+                                 {:db/id (:db/id block)
+                                  :block/children reindex}]]]
+          [:dispatch [:editing/uid new-uid]]]}))
```

Comment:
This does exactly the same thing as `split-block` except it sets the `new-block` as a child to `block`.

Reply:
I actually just implemented adding a new child in case that is more similar than `split-block`, see: https://github.com/athensresearch/athens/pull/366/files#diff-00d35e0f4787abcb3c17f4ebc0ce56b8R527-R537

Some orders are off:
- `new-block`'s should be `0`
- `(inc-after (:db/id block) (:block/order block))` should be `(inc-after (:db/id block) -1)`